barretenberg/ts/src/barretenberg_wasm/barretenberg_wasm_main/index.ts

```diff
@@ -32,7 +32,7 @@ export class BarretenbergWasmMain extends BarretenbergWasmBase {
     module: WebAssembly.Module,
     threads = Math.min(getNumCpu(), BarretenbergWasmMain.MAX_THREADS),
     logger: (msg: string) => void = debug,
-    initial = 25,
+    initial = 26,
```

Comment:
why did this get changed?

Reply:
circleci fails with it set to 25 

https://app.circleci.com/pipelines/github/AztecProtocol/aztec-packages/19242/workflows/f1c0783a-9556-4dd3-a475-29ed8baa3ae1/jobs/860697